### ISSUE TITLE
Single capitalized words are not always translated in the translation pop-up

### ISSUE THREAD

**Issue by acornestean** [mozilla/firefox-translations#675]
### Describe the bug

In the case of single capitalized words added in the input field of the translation pop-up the word is not translated.

This behavior occurs for several target languages and differs based on the input word. For example, using “Apple” and “Beer” as input words, in some languages the word is translated in others it is not.

Adding an article to the word or inputting the word in lower-case does not cause the issue.

The issue occurs on both Firefox for desktop and Firefox for Android.

Example:

English to Czech
- Apple → Apple
- apple → jablko
- The apple → jablko

### To Reproduce

1. Install Firefox Translations v1.3.2
2. Open the translation pop-up and select for example, English as the source language and Czech as the target language
3. In the input field, write “Apple” and notice the translation comes out as “Apple” i.e. the word is not actually translated
4. In the input field, write “apple” and notice the translation comes out as “jablko” i.e. the word is actually translated
5. In the input field, write “The apple” and notice the translation comes out as “jablko” i.e. the word is actually translated

### Expected behavior

Single capitalized words should be translated

### Screenshots

https://user-images.githubusercontent.com/50236075/216571213-d781bf1e-9678-49c0-8a12-f370cf13bfd7.mp4

### System type

None

**Comment by lsomm** [comment]
While I can confirm this behaviour, it seems connected to the fact that it inteprets it as a name, i.e. the company Apple and "Beer" as a name of someone specific(?).

**Comment by marco-c** [comment]
Yeah I think without context it is impossible to know what the actual meaning is.
